server/backend/opendistro_security_client.ts

```diff
@@ -171,9 +175,9 @@ export class SecurityClient {
           };
         }
         throw Error('failed parsing SAML config');
-      } catch (error) {
-        console.log(error);
-        throw new Error();
+      } catch (parsingError) {
+        console.log(parsingError);
```

Comment:
Let's consume kibana's logger so the logs could actually be part of log file.

Reply:
Yeah, agreed, I plan to change all console.log to Kibana logger. Had `TODO` comments in other places. I will create a task in future sprints to fix this after all auth type implementation is complete